### PR TITLE
i18n(ko-KR): add `turso.mdx`

### PR DESCRIPTION
**Pull request by jsparkdev**
#### Description (required)

add `turso.mdx`

#### Related issues & labels (optional)

- Closes #
- Suggested label:

### CODE REVIEW

File: src/content/docs/ko/guides/backend/turso.mdx [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+
+다음 예시는 `WHERE`를 사용하여 `Astro.params`에서 검색된 값이 `slug`인 `posts` 테이블의 항목을 가져온 다음 게시물의 제목을 표시합니다.
+
+```astro title="src/pages/index.astro"
+---
+import {turso} from '../../turso'
+
+const {slug} = Astro.params
+
+const {rows} = await turso.execute({
```
Comment (VoxelMC) [inline src/content/docs/ko/guides/backend/turso.mdx:115]:
```suggestion
import { turso } from '../../turso'

const { slug } = Astro.params

const { rows } = await turso.execute({
```

File: src/content/docs/ko/guides/backend/turso.mdx [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+
+데이터베이스의 정보에 액세스하려면 `turso`를 가져오고 `.astro` 컴포넌트에서 [SQL 쿼리를 실행](https://docs.turso.tech/libsql/client-access/javascript-typescript-sdk#execute-a-single-statement)하세요.
+
+다음 예시는 테이블에서 모든 `posts`를 가져온 다음 `<BlogIndex />` 컴포넌트에 제목 목록을 표시합니다.
+
+```astro title="src/components/BlogIndex.astro"
+---
+import {turso} from '../../turso'
+
+const {rows} = await turso.execute('SELECT * FROM posts')
```
Comment (VoxelMC) [inline src/content/docs/ko/guides/backend/turso.mdx:93]:
```suggestion
import { turso } from '../../turso'

const { rows } = await turso.execute('SELECT * FROM posts')
```

**Review by VoxelMC — COMMENTED** [review]
Thank you for updating this! One step closer :)
I added some comments which will resolve some formatting errors in the original `en` version of the file (#6162). I recommend that this PR be merged after #6162 to avoid any confusion with the i18n detector. 

Once accepted, `As a non-native speaker, I reviewed this PR using the Docs Translation Review Checklist and it LGTM!`

File: src/content/docs/ko/guides/backend/turso.mdx (hunk 2 above)
Comment (sarah11918) [reply src/content/docs/ko/guides/backend/turso.mdx:93]: I was just coming in this PR to ask you to make these suggestions, @VoxelMC !  :purple_heart:

**Comment by jsparkdev** [comment]
Thanks for the new update! @sarah11918  @VoxelMC

**Review by sarah11918 — APPROVED** [review]
:muscle:

**Pull request merged**